cg/meta/rsync/rsync_api.py

```diff
@@ -201,13 +201,19 @@ def slurm_rsync_single_case(
             sample_files_present=sample_files_present,
             case_files_present=case_files_present,
         )
+        existing_folder_list: List[str] = [
+            folder
+            for folder in folder_list
+            if (source_and_destination_paths["delivery_source_path"].joinpath(folder).exists())
+        ]
         commands: str = RsyncAPI.concatenate_rsync_commands(
-            folder_list=folder_list,
+            folder_list=existing_folder_list,
             source_and_destination_paths=source_and_destination_paths,
             ticket=ticket,
         )
-
-        return self.sbatch_rsync_commands(commands=commands, job_prefix=case_id, dry_run=dry_run)
+        return folder_list == existing_folder_list, self.sbatch_rsync_commands(
+            commands=commands, job_prefix=case_id, dry_run=dry_run
+        )
```

Comment:
```suggestion
       is_complete_delivery: bool = folder_list == existing_folder_list
        return is_complete_delivery, self.sbatch_rsync_commands(
            commands=commands, job_prefix=case_id, dry_run=dry_run
        )
```
A bit longer but I think it is more readable and easier to understand

Reply:
Agreed. Adressed in #1650